### ISSUE TITLE
Does not work with KMP 1.4.20

### ISSUE THREAD

**Issue by sebarthel89** [ge-org/multiplatform-swiftpackage#8]
Hi,

thanks for this plugin, it looks exactly what I am looking for.

Unfortunately, I can't get the tasks run successfully with a fresh KMP project version 1.4.20.

My settings:
```
multiplatformSwiftPackage {
    swiftToolsVersion("5.3")
    targetPlatforms {
        iOS { v("13") }
    }
}
```

I get 2 errros:

1. Package name is invalid: null

```
* What went wrong:
A problem occurred configuring root project 'centralized-key-value-library'.
> Could not create task ':createXCFramework'.
   > * Package name is invalid: null
       Either declare the base name of your frameworks or use a non-empty package name.

```

I can fix this by setting it manually:

```
multiplatformSwiftPackage {
    swiftToolsVersion("5.3")
    targetPlatforms {
        iOS { v("13") }
    }
    packageName(project.name)
}
```

2.  at least one framework or library must be specified.

```
> Task :createXCFramework FAILED
error: at least one framework or library must be specified
```

I could not fix this one on my own. Maybe some internal structure has been changed?

**Comment by ge-org** [comment]
Hi @sebarthel89,

could you share the entire `build.gradle` file? Or at least the parts where the KMP project is configured?

**Comment by sebarthel89** [comment]
Sure, here is a minimal example:
https://github.com/sebarthel89/example-kmp-swift-package

**Comment by ge-org** [comment]
Hey @sebarthel89,

thanks again for the bug report and the sample project.

I published version 2.0.2 which contains a fix. This should also allow you to get rid of the `packageName(project.name)` config.

It will take a couple of hours for the release to show up on Maven Central. In the meantime you can use version `2.0.2-SNAPSHOT`.

**Comment by sebarthel89** [comment]
Works, very cool, thank you! 

I will create another issue to target a fat framework issue.
